footer.php

```diff
@@ -41,7 +41,7 @@
 
 						$credit_text = apply_filters(
 							'siteorigin_corp_footer_credits',
-							sprintf( esc_html__( 'Crafted with love by %s.', 'siteorigin-corp' ), '<a href="https://siteorigin.com/" rel="designer">SiteOrigin</a>' )
```

Comment:
Why'd we remove `rel="designer"`?

Reply:
Following _s lead here.

Reply:
Here is their commit: https://github.com/Automattic/_s/commit/7a04605a0b78c7b530914e327e82a4175bafdafa

And the discussions:
https://github.com/Automattic/_s/issues/1016
https://github.com/Automattic/_s/pull/1020